### ISSUE TITLE
🗃️ Build "revise term" alert box

### ISSUE THREAD

**Issue by tatianamac** [tatianamac/selfdefined#141]
## Overview
Build a component that follows each definition that encourages readers to submit revisions if needed.

Recommendation would be to build a separate `.njk` block that can be called by the definition template.

## Design
Threw together a rough wireframe. It's not in the correct fonts but should give you an idea of the general structure.

Reach out with any questions!

**Comment by tatianamac** [comment]
@jlealess Here is an issue for you!

**Comment by tatianamac** [comment]
@Jlealess Let me know if you have questions around this and/or are still able to contribute! I've opened up Self-Defined Saturdays on our Slack channel so folks have resources to help.

**Comment by ovlb** [comment]
@tatianamac It might be nice to have a «Edit this page on GitHub» link inside of the box. I think that’s a good short cut, especially if folks only want to fix a typo or something.

**Comment by tatianamac** [comment]
Agreed!

**Comment by jlealess** [comment]
@tatianamac Yes, I'm definitely still around and looking to contribute! I started working on this last weekend and am hoping to finish up this weekend; a shortage of good 11ty guides has slowed me down but I'll have something to review soon and will add the link to edit on GitHub in the box.
